### DISCUSSION TITLE
Issue with styles in GrapesJS Preset Webpage

### DISCUSSION THREAD

**Discussion by Ananthumpillai** [GrapesJS/grapesjs#5916] [Q&A] [answered]
If I drop two Text Sections to the canvas make changes to the first one, the changes are reflected on the second one as well.
Is there any solution for this?

As per my understanding, all text sections are having common classname ''paragraph" and "heading".

[screencast-from-29-05-24-113446-am-ist_E81lWWfa.webm](https://github.com/GrapesJS/grapesjs/assets/138502626/fbced838-d249-436a-8a10-f7e94f783ea1)

**Comment by artf** [comment, marked as answer]
https://grapesjs.com/docs/modules/Selectors.html#component-first-selectors
